apps/docusaurus/docs/guides/system-integrators-guide.md

```diff
@@ -180,6 +180,36 @@ The REST API offers querying transactions and events in these ways:
 - [Transaction by version](https://api.devnet.aptoslabs.com/v1/spec#/operations/get_transaction_by_version)
 - [Events by event handle](https://api.devnet.aptoslabs.com/v1/spec#/operations/get_events_by_event_handle)
 
+## Exchanging and tracking fungible assets
+
+Aptos has a standard [Fungible Asset](https://github.com/aptos-labs/aptos-core/blob/main/aptos-move/framework/aptos-framework/sources/fungible_asset.move). Different types of fungible asset (FA) can be represented in this standard through the use of distinct metadata object.
```

Comment:
Oh boy the system integrators guide, we need to rewrite this whole thing at some point